grafeas-charts/templates/NOTES.txt

```diff
@@ -0,0 +1,19 @@
+1. The Grafeas server can be accessed via port {{ .Values.service.port }} on the following DNS name from within your cluster:
```

Comment:
I'm not familiar with the idiomatic usage in this space: will the templated values be filled in when displayed to the user, or will they appear as raw `{{ .Values....}}`?

Reply:
You can find some examples in the main stable charts repo e.g. https://github.com/helm/charts/blob/master/stable/grafana/templates/NOTES.txt

Reply:
Perfect, thanks for linking the example.